Q/Qt6Base/build_tarballs.jl

```diff
@@ -28,31 +28,51 @@ commonoptions=" \
 -openssl-linked  -nomake examples -release \
 "
 
+commoncmakeoptions="-DCMAKE_PREFIX_PATH=${prefix} -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DQT_HOST_PATH=$host_prefix -DQT_FEATURE_openssl_linked=ON"
+
+export LD_LIBRARY_PATH=$host_libdir:$LD_LIBRARY_PATH
 export OPENSSL_LIBS="-L${libdir} -lssl -lcrypto"
 
 sed -i 's/-march=haswell/-mavx2/' $qtsrcdir/cmake/QtCompilerOptimization.cmake
+sed -i 's/.*Ws2_32 Crypt32.*//' $qtsrcdir/cmake/FindWrapOpenSSL.cmake
+sed -i "s!Qt::GuiPrivate!Qt::GuiPrivate $libdir/clang/11.0.1/lib/darwin/libclang_rt.osx.a!" $qtsrcdir/src/plugins/platforms/cocoa/CMakeLists.txt
 
 case "$target" in
 
     x86_64-linux-musl*)
         ../qtbase-everywhere-src-*/configure -prefix $prefix $commonoptions -fontconfig -- -DCMAKE_PREFIX_PATH=${prefix} -DCMAKE_TOOLCHAIN_FILE=${CMAKE_HOST_TOOLCHAIN}
     ;;
+
+    *mingw*)
+        ../qtbase-everywhere-src-*/configure -prefix $prefix $commonoptions -opengl dynamic -- $commoncmakeoptions
+    ;;
+
+    *apple-darwin*)
+        # Install 11.1 SDK
+        cd $WORKSPACE/srcdir/MacOSX11.1.sdk
+        rm -rf /opt/$target/$target/sys-root/System
+        rm -rf /opt/$target/$target/sys-root/usr/include/libxml2/
+        rsync -a usr/* /opt/$target/$target/sys-root/usr/
+        cp -a System /opt/$target/$target/sys-root/
+        cd $WORKSPACE/srcdir/build
+
+        ../qtbase-everywhere-src-*/configure -prefix $prefix $commonoptions -- $commoncmakeoptions -DCMAKE_FRAMEWORK_PATH=/opt/$target/$target/sys-root/System/Library/Frameworks -DCMAKE_OSX_DEPLOYMENT_TARGET=10.14 -DCUPS_INCLUDE_DIR=/opt/$target/$target/sys-root/usr/include -DCUPS_LIBRARIES=/opt/$target/$target/sys-root/usr/lib/libcups.tbd
+    ;;
+
+    *)
+        ../qtbase-everywhere-src-*/configure -prefix $prefix $commonoptions -fontconfig -- $commoncmakeoptions
+    ;;
 esac
 
 cmake --build . --parallel
```

Comment:
Can we specify the number of parallel jobs?  I think this is hammering the build machine

Reply:
Yes, will do :)